std_msgs/Header header
string accid
string sw_version
string imu_status
string camera_status
string motor_status
string battery
string status
